client/components/Alert/Alert.js

```diff
@@ -25,13 +26,11 @@ export function Alert(props) {
 			classname = styles.message;
 		break;
 	}
-
-
-
-
-	return (<div className={[styles.alert, classname].join(" ")}><span>{props.text}</span></div>);
-  
+		return (<div id = "alert" className={[styles.alert, classname].join(" ") } onClick = { () => wasClicked() }><span>{props.text}</span></div>);	
 }
 
+function wasClicked(){
+	document.getElementById("alert").remove();
```

Comment:
change to `document.getElementById("alert").setAttribute("hidden", "true")`

Due to how React works you can't remove objects from the DOM like this.